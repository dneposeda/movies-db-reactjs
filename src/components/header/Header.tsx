import React from "react";
import Logo from "../logo/Logo";

const Header = () => (
    <div className="header">
        <Logo />
    </div>
)

export default Header;
