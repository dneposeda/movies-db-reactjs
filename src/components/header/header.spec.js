import React from 'react';
import Header from './Header';

describe('Header', () => {
    it('renders correctly', () => {
        const component = mount (
            <Header />
        );
        expect(component).toMatchSnapshot();
        expect(component.find('.logo')).toHaveLength(1);
    });
});
