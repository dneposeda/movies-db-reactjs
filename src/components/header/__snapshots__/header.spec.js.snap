// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<Header>
  <div
    className="header"
  >
    <Logo>
      <div
        className="logo"
      >
        <a
          className="logo__link"
          href="/"
        >
          <span
            className="logo__text"
          >
            <span>
              netflix
            </span>
            roulette
          </span>
        </a>
      </div>
    </Logo>
  </div>
</Header>
`;
